lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -163,6 +163,42 @@ ::mlir::LogicalResult mlir::tt::ttir::MatmulOp::verify() {
 }
 // ANCHOR_END: adding_an_op_matmul_ttir_verify
 
+::mlir::LogicalResult mlir::tt::ttir::Conv2dOp::verify() {
```

Comment:
Are we aware of more tt-nn-related constraints? E.g. wight layout, supported attributes, etc.